.gitignore

```diff
@@ -28,3 +28,5 @@
 /libs
 
 /src/main/generated/java
+
+logs/

```

Comment:
Should be `/logs/`, also I have never seen this folder so not sure how you manage to get it :D

Reply:
I got those too

Reply:
Weird..